app/routes/single-business-claim-expired.js

```diff
@@ -1,6 +1,8 @@
 const config = require('../config/index')
 
-module.exports = {
+// is this file/route needed? it doesnt get added to the router
```

Comment:
Probably not if not registered there. I have a feeling this was an old thing from the days of registering for eligibility